string login_name
string password

---

bool valid					# true if input password matches to commander's registered password, otherwise it is false
string message
mrta_vc/Agent voice_commander



